docs/dev.rst

```diff
@@ -14,37 +14,31 @@ All code files need to start with the MPLv2 header::
     # License, v. 2.0. If a copy of the MPL was not distributed with this
     # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-Black
------
+Linting
```

Comment:
I reduced this chapter down to be more flow-centric. As a developer, I want to lint all the things and fix linting issues. It's nice to know what tools are involved, but I really want to know how to do those two tasks. This adjusts the docs accordingly.